elements/pl-threejs/info.json

```diff
@@ -3,15 +3,15 @@
     "dependencies": {
         "nodeModulesScripts": [
             "three/build/three.min.js",
```

Comment:
We're getting a warning that they'll be removing the pre-built threejs files soon in favor of ESM modules. This makes sense from a moving-the-ecosystem-forward perspective, but is going to cause us some pain 😢